rcljava/src/main/cpp/org_ros2_rcljava_node_NodeImpl.cpp

```diff
@@ -309,3 +310,66 @@ Java_org_ros2_rcljava_node_NodeImpl_nativeGetTopicNamesAndTypes(
     rcljava_throw_rclexception(env, ret, "failed to fini topic names and types structure");
   }
 }
+
+JNIEXPORT void JNICALL
+Java_org_ros2_rcljava_node_NodeImpl_nativeGetPublishersInfo(
+  JNIEnv * env, jclass, jlong handle, jstring jtopic_name, jobject jpublishers_info)
+{
+  rcl_node_t * node = reinterpret_cast<rcl_node_t *>(handle);
+  if (!node) {
+    rcljava_throw_exception(
+      env, "java/lang/IllegalStateException", "passed node handle is NULL");
```

Comment:
nit: I think this should be `IllegalArgumentException` (like other cases where we check for null handles).

Reply:
Done in https://github.com/osrf/ros2_java/pull/26/commits/73fba78b06322b03463109df756dbe2f16c778ec.